### ISSUE TITLE
Group by with join

### ISSUE THREAD

**Issue by MarkAntipin** [tortoise/tortoise-orm#372]
**Describe the bug**
I 'm trying to execute sql like

```
SELECT
    u.username
    COUNT(*)
FROM table1 t
JOIN user u ON t.user_id = u.id
GROUP BY u.username;
```

so I have created orm implementation:

```
class User(Model):
    username = CharField(max_length=128, index=True)

    class Meta:
        table = 'user'


class Table1(Model):
    user = ForeignKeyField('models.User', related_name='user_t_id', to_field='id')
    
    class Meta:
        table = 'table1'

record = (await Table1
    .annotate(count=Count('id'))
    .prefetch_related('user')
    .group_by('user__username')
    .values('user__username', 'count')
)
```

And get traceback 
```
Traceback (most recent call last):
  File "/Users/user/tmp.py", line 7, in <module>
    run_async(run())
  File "/Users/user/.venv/lib/python3.7/site-packages/tortoise/__init__.py", line 636, in run_async
    loop.run_until_complete(coro)
  File "/Users/user/.pyenv/versions/3.7.6/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "/Users/user/tmp2.py", line 37, in run
    .values('user__username')
  File "/Users/user/.venv/lib/python3.7/site-packages/tortoise/queryset.py", line 1121, in _execute
    result = await self._db.execute_query_dict(str(self.query))
  File "/Users/user/.venv/lib/python3.7/site-packages/tortoise/backends/asyncpg/client.py", line 38, in translate_exceptions_
    raise OperationalError(exc)
tortoise.exceptions.OperationalError: column table1.user__username does not exist
```

From log I can see that orm is trying to execute that sql, and it is incorrect: 
```
SELECT "table1__user"."username" "user__username"
FROM "table1"
LEFT OUTER JOIN "auth_user" "table1__user" ON "table1__user"."id"="table1"."user_id"
GROUP BY "table1"."user__username"
```

But when I'm doing just join 

```
    record = (
        await Table1
        .all()
        .prefetch_related('user')
        .values('user__username')
    )
```
It works fine.

Can somebody help me with this, maybe I'm doing something wrong.

**Comment by grigi** [comment]
Ah, we never made `.group_by()` join aware.

**Comment by long2ice** [comment]
Thanks to report,will be fixed later.

**Comment by MarkAntipin** [comment]
Thank's guys, i did not expect this to happen so quickly.

**Comment by grigi** [comment]
Hi I released 0.16.10 with the change :smile:
And a pleasure.
Thanks @long2ice for doing the work!
